src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,8 +1,205 @@
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Photo from "../../assets/Group 3.png";
+import "../../App.scss";
+import TextField from "@mui/material/TextField";
+import { Link, useNavigate } from "react-router-dom";
+import { FormControl, Stack } from "@mui/material";
+import { useContext, useState } from "react";
+import { AuthContext } from "../Context/AuthContext/AuthContext";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { toast } from "react-toastify";
+import CircularProgress from "@mui/material/CircularProgress";
+import { IChange } from "./ChangeInterfaces/ChangeInterfaces";
+import KeyboardBackspaceIcon from '@mui/icons-material/KeyboardBackspace';
+import Home from "@mui/icons-material/Home";
+
+
+export default function ChangePassword({ saveAdminData }) {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IChange>();
+  const [isLoading, setIsLoding] = useState(false);
+  const {baseUrl} = useContext(AuthContext);
+  const theme = useTheme();
+  // ****************** to RestPassword **********************
+  const onSubmit = (data: IChange) => {
+    setIsLoding(true);
+    axios
+      .post(`${baseUrl}/admin/users/change-password`, data)
+      .then((response) => {
+       
+        navigate("/login");
+        toast.success("Successfully");
+      })
+      .catch((error) => {
+        console.log(error);
+        toast.error(error);
+        setIsLoding(false);
+      });
+  };
   return (
-    <div>ChangePassword</div>
-  )
-}
+    <>
+      <Stack spacing={30} direction="row" justifyContent="center">
+        <Box sx={{ display: "flex", flexDirection: "column" }}>
+          <CardContent sx={{ flex: "1 0 auto", marginLeft: 5 }}>
+            <Typography
+              sx={{ mb: 1 }}
+              component="div"
+              variant="h5"
+              className="text"
+            >
+              Stay<span className="text2">cation.</span>
+            </Typography>
+            <Box sx={{ marginLeft: 5, }}>
+              <Typography variant="h3" component="div" sx={{ mb: 2 }}>
+               Change Password
+              </Typography>
+              <Typography variant="h6" component="div" className="text3">
+                If you don’t have an account register <br />
+                You can <span>Login here !</span>
+              </Typography>
+              <FormControl
+                onSubmit={handleSubmit(onSubmit)}
+                component="form"
+                sx={{
+                  "& .MuiTextField-root": {
+                    mt: 4,
+                    width: "35ch",
+                    height: "5ch",
+                  },
+                }}
+                noValidate
+                autoComplete="off"
```

Comment:
autoComplete is beneficial for faster writing and filling the form easily, we should turn it off.